iati/core/tests/test_rulesets.py

```diff
@@ -187,14 +183,38 @@ class RuleSubclassTestBase(object):
     """A base class for Rule subclass tests."""
 
     @pytest.fixture
-    def basic_rule(self, rule_constructor, valid_case):
+    def xpath_base(self):
+        """Return valid xpath_base."""
+        return '//root_element'
+
+    @pytest.fixture
+    def empty_xpath_base(self):
+        """Empty_xpath_base."""
+        return ''
+
+    @pytest.fixture
+    def basic_rule(self, rule_constructor, valid_case, xpath_base):
         """Instantiate a basic Rule subclass."""
-        xpath_base = 'an xpath'
         return rule_constructor(xpath_base, valid_case)
 
     @pytest.fixture
+    def rule(self, rule_constructor, xpath_base, case_for_is_valid_for):
```

Comment:
The name of this should be classified more specifically - `rule_[something-meaningful]`.